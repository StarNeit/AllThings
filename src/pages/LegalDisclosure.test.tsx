import React from 'react'
import LegalDisclosure from 'pages/LegalDisclosure'
import { Provider } from 'react-redux'

describe('Check the legal disclosure component', () => {
  it('should render correctly if the user is not logged in', () => {
    const state = {
      authentication: {
        loggedIn: false,
      },
    }
    const store = global.mockStore(state)
    const wrapper = global.mountIntl(
      <Provider store={store}>
        <LegalDisclosure />
      </Provider>,
    )
    expect(wrapper).toMatchSnapshot()
  })
  it('should render differently if the user is logged in', () => {
    const state = {
      authentication: {
        loggedIn: true,
      },
      app: {
        embeddedLayout: false,
        microApps: [],
      },
    } as any
    const store = global.mockStore(state)
    const wrapper = global.mountIntl(
      <Provider store={store}>
        <LegalDisclosure />
      </Provider>,
    )
    expect(wrapper).toMatchSnapshot()
  })
})
