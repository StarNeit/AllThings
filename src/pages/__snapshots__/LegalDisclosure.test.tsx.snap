// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Check the legal disclosure component should render correctly if the user is not logged in 1`] = `
.css-0,
[data-css-0] {
  margin-bottom: 20px;
}

.css-1,
[data-css-1] {
  margin-right: 10px;
}

.css-2,
[data-css-2] {
  border-radius: 10px;
  padding: 40px;
}

.css-3,
[data-css-3] {
  margin-top: 40px;
}

.css-4,
[data-css-4] {
  box-sizing: border-box;
  flex: 0 0 auto;
  -webkit-flex: 0 0 auto;
}

.css-5,
[data-css-5] {
  height: 50px;
  background-color: transparent;
  position: relative;
  z-index: 2;
}

.css-6,
[data-css-6] {
  box-sizing: border-box;
  align-content: center;
  align-items: center;
  display: -webkit-box;
  display: -moz-box;
  display: -ms-flexbox;
  display: -webkit-flex;
  display: flex;
  flex-direction: row;
  justify-content: flex-start;
  flex: 0 0 auto;
  -webkit-align-content: center;
  -webkit-box-align: center;
  -webkit-align-items: center;
  -webkit-box-orient: horizontal;
  -webkit-box-direction: normal;
  -webkit-flex-direction: row;
  -webkit-box-pack: start;
  -webkit-justify-content: flex-start;
  -webkit-flex: 0 0 auto;
}

.css-7,
[data-css-7] {
  box-sizing: border-box;
  align-content: center;
  align-items: center;
  display: -webkit-box;
  display: -moz-box;
  display: -ms-flexbox;
  display: -webkit-flex;
  display: flex;
  flex-direction: row;
  justify-content: flex-start;
  flex: 0 0 auto;
  cursor: pointer;
  -webkit-align-content: center;
  -webkit-box-align: center;
  -webkit-align-items: center;
  -webkit-box-orient: horizontal;
  -webkit-box-direction: normal;
  -webkit-flex-direction: row;
  -webkit-box-pack: start;
  -webkit-justify-content: flex-start;
  -webkit-flex: 0 0 auto;
}

.css-8,
[data-css-8] {
  height: 50px;
  width: 50px;
  border-radius: 2px;
  background-color: transparent;
}

.css-9,
[data-css-9] {
  box-sizing: border-box;
  align-content: center;
  align-items: center;
  display: -webkit-box;
  display: -moz-box;
  display: -ms-flexbox;
  display: -webkit-flex;
  display: flex;
  flex-direction: row;
  justify-content: center;
  flex: 0 0 auto;
  -webkit-align-content: center;
  -webkit-box-align: center;
  -webkit-align-items: center;
  -webkit-box-orient: horizontal;
  -webkit-box-direction: normal;
  -webkit-flex-direction: row;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -webkit-flex: 0 0 auto;
}

.css-10,
[data-css-10] {
  width: 16px;
  height: 16px;
  fill: lightBlack;
}

.css-11,
[data-css-11] {
  display: block;
  font-family: "Open Sans", Helvetica, Arial, sans-serif;
  font-weight: 600;
  font-size: 14px;
  -webkit-font-smoothing: antialiased;
  -moz-osx-font-smoothing: grayscale;
  color: #333333;
}

.css-12,
[data-css-12] {
  box-sizing: border-box;
  align-content: center;
  align-items: center;
  display: -webkit-box;
  display: -moz-box;
  display: -ms-flexbox;
  display: -webkit-flex;
  display: flex;
  flex-direction: column;
  justify-content: flex-start;
  flex: 0 0 auto;
  -webkit-align-content: center;
  -webkit-box-align: center;
  -webkit-align-items: center;
  -webkit-box-orient: vertical;
  -webkit-box-direction: normal;
  -webkit-flex-direction: column;
  -webkit-box-pack: start;
  -webkit-justify-content: flex-start;
  -webkit-flex: 0 0 auto;
}

.css-13,
[data-css-13] {
  display: block;
  font-family: "Open Sans", Helvetica, Arial, sans-serif;
  font-weight: 600;
  font-size: 14px;
  text-align: center;
  -webkit-font-smoothing: antialiased;
  -moz-osx-font-smoothing: grayscale;
  color: #333333;
}

.css-14,
[data-css-14] {
  display: block;
  font-family: "Open Sans", Helvetica, Arial, sans-serif;
  font-weight: 400;
  font-size: 12px;
  text-align: center;
  -webkit-font-smoothing: antialiased;
  -moz-osx-font-smoothing: grayscale;
  color: #333333;
}

.css-15,
[data-css-15] {
  box-sizing: border-box;
  align-content: stretch;
  align-items: stretch;
  display: -webkit-box;
  display: -moz-box;
  display: -ms-flexbox;
  display: -webkit-flex;
  display: flex;
  flex-direction: row;
  justify-content: center;
  flex: 0 0 auto;
  -webkit-align-content: stretch;
  -webkit-box-align: stretch;
  -webkit-align-items: stretch;
  -webkit-box-orient: horizontal;
  -webkit-box-direction: normal;
  -webkit-flex-direction: row;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -webkit-flex: 0 0 auto;
}

.css-16,
[data-css-16] {
  width: 21.5px;
  height: 21.5px;
  stroke: #7e8c8d;
}

.css-17,
[data-css-17] {
  display: block;
  font-family: "Open Sans", Helvetica, Arial, sans-serif;
  font-weight: 400;
  font-size: 14px;
  -webkit-font-smoothing: antialiased;
  -moz-osx-font-smoothing: grayscale;
  color: #333333;
}

@media (max-width: 640px) {
  .css-8,
  [data-css-8] {
    height: 40px;
    width: 40px;
  }
}

<Connect(injectIntl(LegalDisclosure))>
  <ForwardRef>
    <div
      data-css-4=""
    >
      <ForwardRef
        data-css-0=""
      >
        <div
          data-css-0=""
          data-css-4=""
        >
          <ForwardRef
            alignV="center"
            data-css-5=""
            direction="row"
          >
            <div
              data-css-6=""
              data-css-5=""
            >
              <ForwardRef
                alignV="center"
                data-e2e="back-button"
                direction="row"
                onClick={[Function]}
              >
                <div
                  data-css-7=""
                  data-e2e="back-button"
                  onClick={[Function]}
                >
                  <ForwardRef
                    alignH="center"
                    alignV="center"
                    data-css-8=""
                    direction="row"
                  >
                    <div
                      data-css-9=""
                      data-css-8=""
                    >
                      <ForwardRef
                        alignH="center"
                        alignV="center"
                        dangerouslySetInnerHTML={
                          Object {
                            "__html": "",
                          }
                        }
                        data-css-10=""
                      >
                        <div
                          dangerouslySetInnerHTML={
                            Object {
                              "__html": "",
                            }
                          }
                          data-css-4=""
                          data-css-10=""
                        />
                      </ForwardRef>
                    </div>
                  </ForwardRef>
                  <ForwardRef
                    data-css-11=""
                  >
                    <div
                      data-css-11=""
                      data-css-4=""
                    >
                      <span>
                        Back
                      </span>
                    </div>
                  </ForwardRef>
                </div>
              </ForwardRef>
            </div>
          </ForwardRef>
          <ForwardRef
            alignV="center"
            data-css-2=""
            direction="column"
          >
            <div
              data-css-2=""
              data-css-12=""
            >
              <ForwardRef
                key="Allthings Technologies AG"
              >
                <div
                  data-css-4=""
                >
                  <ForwardRef
                    data-css-13=""
                  >
                    <div
                      data-css-13=""
                      data-css-4=""
                    >
                      Allthings Technologies AG
                    </div>
                  </ForwardRef>
                  <ForwardRef
                    data-css-14=""
                  >
                    <div
                      data-css-14=""
                      data-css-4=""
                    >
                      Lange Gasse 8
                    </div>
                  </ForwardRef>
                  <ForwardRef
                    data-css-14=""
                  >
                    <div
                      data-css-14=""
                      data-css-4=""
                    >
                      CH-4052 Basel
                    </div>
                  </ForwardRef>
                  <ForwardRef
                    data-css-14=""
                  >
                    <div
                      data-css-14=""
                      data-css-4=""
                    >
                      Schweiz
                    </div>
                  </ForwardRef>
                </div>
              </ForwardRef>
              <ForwardRef
                data-css-3=""
                key="Allthings GmbH"
              >
                <div
                  data-css-3=""
                  data-css-4=""
                >
                  <ForwardRef
                    data-css-13=""
                  >
                    <div
                      data-css-13=""
                      data-css-4=""
                    >
                      Allthings GmbH
                    </div>
                  </ForwardRef>
                  <ForwardRef
                    data-css-14=""
                  >
                    <div
                      data-css-14=""
                      data-css-4=""
                    >
                      Unicorn Village
                    </div>
                  </ForwardRef>
                  <ForwardRef
                    data-css-14=""
                  >
                    <div
                      data-css-14=""
                      data-css-4=""
                    >
                      Richardstra√üe 85
                    </div>
                  </ForwardRef>
                  <ForwardRef
                    data-css-14=""
                  >
                    <div
                      data-css-14=""
                      data-css-4=""
                    >
                      D-12043 Berlin
                    </div>
                  </ForwardRef>
                  <ForwardRef
                    data-css-14=""
                  >
                    <div
                      data-css-14=""
                      data-css-4=""
                    >
                      Deutschland
                    </div>
                  </ForwardRef>
                </div>
              </ForwardRef>
              <ForwardRef
                data-css-3=""
                key="Represented by"
              >
                <div
                  data-css-3=""
                  data-css-4=""
                >
                  <ForwardRef
                    data-css-13=""
                  >
                    <div
                      data-css-13=""
                      data-css-4=""
                    >
                      Represented by
                    </div>
                  </ForwardRef>
                  <ForwardRef
                    data-css-14=""
                  >
                    <div
                      data-css-14=""
                      data-css-4=""
                    >
                      Stefan Zanetti
                    </div>
                  </ForwardRef>
                </div>
              </ForwardRef>
              <ForwardRef
                data-css-3=""
                key="Contact"
              >
                <div
                  data-css-3=""
                  data-css-4=""
                >
                  <ForwardRef
                    data-css-13=""
                  >
                    <div
                      data-css-13=""
                      data-css-4=""
                    >
                      Contact
                    </div>
                  </ForwardRef>
                  <ForwardRef
                    alignH="center"
                    direction="row"
                    key="email"
                  >
                    <div
                      data-css-15=""
                    >
                      <ForwardRef
                        alignH="center"
                        alignV="center"
                        dangerouslySetInnerHTML={
                          Object {
                            "__html": "",
                          }
                        }
                        data-css-16=""
                        data-css-1=""
                      >
                        <div
                          dangerouslySetInnerHTML={
                            Object {
                              "__html": "",
                            }
                          }
                          data-css-16=""
                          data-css-4=""
                          data-css-1=""
                        />
                      </ForwardRef>
                      <a
                        href="mailto:support@allthings.me"
                      >
                        support@allthings.me
                      </a>
                    </div>
                  </ForwardRef>
                  <ForwardRef
                    alignH="center"
                    direction="row"
                    key="phone"
                  >
                    <div
                      data-css-15=""
                    >
                      <ForwardRef
                        alignH="center"
                        alignV="center"
                        dangerouslySetInnerHTML={
                          Object {
                            "__html": "",
                          }
                        }
                        data-css-16=""
                        data-css-1=""
                      >
                        <div
                          dangerouslySetInnerHTML={
                            Object {
                              "__html": "",
                            }
                          }
                          data-css-16=""
                          data-css-4=""
                          data-css-1=""
                        />
                      </ForwardRef>
                      <ForwardRef
                        data-css-17=""
                      >
                        <div
                          data-css-4=""
                          data-css-17=""
                        >
                          +49 1511 003 22 54
                        </div>
                      </ForwardRef>
                    </div>
                  </ForwardRef>
                  <ForwardRef
                    alignH="center"
                    direction="row"
                    key="twitter"
                  >
                    <div
                      data-css-15=""
                    >
                      <svg
                        data-css-1=""
                        height="21px"
                        stroke="#7e8c8d"
                        viewBox="0 0 24 24"
                        width="21px"
                      >
                        <path
                          d="M23.407,4.834c-0.814,0.363 -1.5,0.375 -2.228,0.016c0.938,-0.562 0.981,-0.957 1.32,-2.019c-0.878,0.521 -1.851,0.9 -2.886,1.104c-0.827,-0.882 -2.009,-1.435 -3.315,-1.435c-2.51,0 -4.544,2.036 -4.544,4.544c0,0.356 0.04,0.703 0.117,1.036c-3.776,-0.189 -7.125,-1.998 -9.366,-4.748c-0.391,0.671 -0.615,1.452 -0.615,2.285c0,1.577 0.803,2.967 2.021,3.782c-0.745,-0.024 -1.445,-0.228 -2.057,-0.568c-0.001,0.019 -0.001,0.037 -0.001,0.057c0,2.202 1.566,4.038 3.646,4.456c-0.666,0.181 -1.368,0.209 -2.053,0.079c0.579,1.804 2.257,3.118 4.245,3.155c-1.944,1.524 -4.355,2.159 -6.728,1.881c2.012,1.289 4.399,2.041 6.966,2.041c8.358,0 12.928,-6.924 12.928,-12.929c0,-0.198 -0.003,-0.393 -0.012,-0.588c0.886,-0.64 1.953,-1.237 2.562,-2.149Z"
                          fill="none"
                          strokeLinejoin="round"
                          strokeWidth="1"
                        />
                      </svg>
                      <a
                        href="https://twitter.com/allthings_here"
                        rel="noopener"
                        target="_blank"
                      >
                        allthings_here
                      </a>
                    </div>
                  </ForwardRef>
                </div>
              </ForwardRef>
              <ForwardRef
                data-css-3=""
                key="Trade register entry"
              >
                <div
                  data-css-3=""
                  data-css-4=""
                >
                  <ForwardRef
                    data-css-13=""
                  >
                    <div
                      data-css-13=""
                      data-css-4=""
                    >
                      Trade register entry
                    </div>
                  </ForwardRef>
                  <ForwardRef
                    data-css-14=""
                  >
                    <div
                      data-css-14=""
                      data-css-4=""
                    >
                      Register office: Basel
                    </div>
                  </ForwardRef>
                  <ForwardRef
                    data-css-14=""
                  >
                    <div
                      data-css-14=""
                      data-css-4=""
                    >
                      Register number: CH-270.3.014.814-4
                    </div>
                  </ForwardRef>
                </div>
              </ForwardRef>
              <ForwardRef
                data-css-3=""
                key="Tax number (Switzerland)"
              >
                <div
                  data-css-3=""
                  data-css-4=""
                >
                  <ForwardRef
                    data-css-13=""
                  >
                    <div
                      data-css-13=""
                      data-css-4=""
                    >
                      Tax number (Switzerland)
                    </div>
                  </ForwardRef>
                  <ForwardRef
                    data-css-14=""
                  >
                    <div
                      data-css-14=""
                      data-css-4=""
                    >
                      CHE-191.384.657 MWST
                    </div>
                  </ForwardRef>
                </div>
              </ForwardRef>
            </div>
          </ForwardRef>
        </div>
      </ForwardRef>
    </div>
  </ForwardRef>
</Connect(injectIntl(LegalDisclosure))>
`;

exports[`Check the legal disclosure component should render differently if the user is logged in 1`] = `
.css-0,
[data-css-0] {
  margin-bottom: 20px;
}

.css-1,
[data-css-1] {
  margin-right: 10px;
}

.css-2,
[data-css-2] {
  border-radius: 10px;
  padding: 40px;
}

.css-3,
[data-css-3] {
  margin-top: 40px;
}

.css-4,
[data-css-4] {
  box-sizing: border-box;
  flex: 0 0 auto;
  -webkit-flex: 0 0 auto;
}

.css-5,
[data-css-5] {
  height: 50px;
  background-color: transparent;
  position: relative;
  z-index: 2;
}

.css-6,
[data-css-6] {
  box-sizing: border-box;
  align-content: center;
  align-items: center;
  display: -webkit-box;
  display: -moz-box;
  display: -ms-flexbox;
  display: -webkit-flex;
  display: flex;
  flex-direction: row;
  justify-content: flex-start;
  flex: 0 0 auto;
  -webkit-align-content: center;
  -webkit-box-align: center;
  -webkit-align-items: center;
  -webkit-box-orient: horizontal;
  -webkit-box-direction: normal;
  -webkit-flex-direction: row;
  -webkit-box-pack: start;
  -webkit-justify-content: flex-start;
  -webkit-flex: 0 0 auto;
}

.css-7,
[data-css-7] {
  box-sizing: border-box;
  align-content: center;
  align-items: center;
  display: -webkit-box;
  display: -moz-box;
  display: -ms-flexbox;
  display: -webkit-flex;
  display: flex;
  flex-direction: row;
  justify-content: flex-start;
  flex: 0 0 auto;
  cursor: pointer;
  -webkit-align-content: center;
  -webkit-box-align: center;
  -webkit-align-items: center;
  -webkit-box-orient: horizontal;
  -webkit-box-direction: normal;
  -webkit-flex-direction: row;
  -webkit-box-pack: start;
  -webkit-justify-content: flex-start;
  -webkit-flex: 0 0 auto;
}

.css-8,
[data-css-8] {
  height: 50px;
  width: 50px;
  border-radius: 2px;
  background-color: transparent;
}

.css-9,
[data-css-9] {
  box-sizing: border-box;
  align-content: center;
  align-items: center;
  display: -webkit-box;
  display: -moz-box;
  display: -ms-flexbox;
  display: -webkit-flex;
  display: flex;
  flex-direction: row;
  justify-content: center;
  flex: 0 0 auto;
  -webkit-align-content: center;
  -webkit-box-align: center;
  -webkit-align-items: center;
  -webkit-box-orient: horizontal;
  -webkit-box-direction: normal;
  -webkit-flex-direction: row;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -webkit-flex: 0 0 auto;
}

.css-10,
[data-css-10] {
  width: 16px;
  height: 16px;
  fill: lightBlack;
}

.css-11,
[data-css-11] {
  display: block;
  font-family: "Open Sans", Helvetica, Arial, sans-serif;
  font-weight: 600;
  font-size: 14px;
  -webkit-font-smoothing: antialiased;
  -moz-osx-font-smoothing: grayscale;
  color: #333333;
}

.css-12,
[data-css-12] {
  box-sizing: border-box;
  align-content: center;
  align-items: center;
  display: -webkit-box;
  display: -moz-box;
  display: -ms-flexbox;
  display: -webkit-flex;
  display: flex;
  flex-direction: column;
  justify-content: flex-start;
  flex: 0 0 auto;
  -webkit-align-content: center;
  -webkit-box-align: center;
  -webkit-align-items: center;
  -webkit-box-orient: vertical;
  -webkit-box-direction: normal;
  -webkit-flex-direction: column;
  -webkit-box-pack: start;
  -webkit-justify-content: flex-start;
  -webkit-flex: 0 0 auto;
}

.css-13,
[data-css-13] {
  display: block;
  font-family: "Open Sans", Helvetica, Arial, sans-serif;
  font-weight: 600;
  font-size: 14px;
  text-align: center;
  -webkit-font-smoothing: antialiased;
  -moz-osx-font-smoothing: grayscale;
  color: #333333;
}

.css-14,
[data-css-14] {
  display: block;
  font-family: "Open Sans", Helvetica, Arial, sans-serif;
  font-weight: 400;
  font-size: 12px;
  text-align: center;
  -webkit-font-smoothing: antialiased;
  -moz-osx-font-smoothing: grayscale;
  color: #333333;
}

.css-15,
[data-css-15] {
  box-sizing: border-box;
  align-content: stretch;
  align-items: stretch;
  display: -webkit-box;
  display: -moz-box;
  display: -ms-flexbox;
  display: -webkit-flex;
  display: flex;
  flex-direction: row;
  justify-content: center;
  flex: 0 0 auto;
  -webkit-align-content: stretch;
  -webkit-box-align: stretch;
  -webkit-align-items: stretch;
  -webkit-box-orient: horizontal;
  -webkit-box-direction: normal;
  -webkit-flex-direction: row;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -webkit-flex: 0 0 auto;
}

.css-16,
[data-css-16] {
  width: 21.5px;
  height: 21.5px;
  stroke: #7e8c8d;
}

.css-17,
[data-css-17] {
  display: block;
  font-family: "Open Sans", Helvetica, Arial, sans-serif;
  font-weight: 400;
  font-size: 14px;
  -webkit-font-smoothing: antialiased;
  -moz-osx-font-smoothing: grayscale;
  color: #333333;
}

@media (max-width: 640px) {
  .css-8,
  [data-css-8] {
    height: 40px;
    width: 40px;
  }
}

<Connect(injectIntl(LegalDisclosure))>
  <ForwardRef>
    <div
      data-css-4=""
    >
      <ForwardRef
        data-css-0=""
      >
        <div
          data-css-0=""
          data-css-4=""
        >
          <ForwardRef
            alignV="center"
            data-css-5=""
            direction="row"
          >
            <div
              data-css-6=""
              data-css-5=""
            >
              <ForwardRef
                alignV="center"
                data-e2e="back-button"
                direction="row"
                onClick={[Function]}
              >
                <div
                  data-css-7=""
                  data-e2e="back-button"
                  onClick={[Function]}
                >
                  <ForwardRef
                    alignH="center"
                    alignV="center"
                    data-css-8=""
                    direction="row"
                  >
                    <div
                      data-css-9=""
                      data-css-8=""
                    >
                      <ForwardRef
                        alignH="center"
                        alignV="center"
                        dangerouslySetInnerHTML={
                          Object {
                            "__html": "",
                          }
                        }
                        data-css-10=""
                      >
                        <div
                          dangerouslySetInnerHTML={
                            Object {
                              "__html": "",
                            }
                          }
                          data-css-4=""
                          data-css-10=""
                        />
                      </ForwardRef>
                    </div>
                  </ForwardRef>
                  <ForwardRef
                    data-css-11=""
                  >
                    <div
                      data-css-11=""
                      data-css-4=""
                    >
                      <span>
                        Back
                      </span>
                    </div>
                  </ForwardRef>
                </div>
              </ForwardRef>
            </div>
          </ForwardRef>
          <ForwardRef
            alignV="center"
            data-css-2=""
            direction="column"
          >
            <div
              data-css-2=""
              data-css-12=""
            >
              <ForwardRef
                key="Allthings Technologies AG"
              >
                <div
                  data-css-4=""
                >
                  <ForwardRef
                    data-css-13=""
                  >
                    <div
                      data-css-13=""
                      data-css-4=""
                    >
                      Allthings Technologies AG
                    </div>
                  </ForwardRef>
                  <ForwardRef
                    data-css-14=""
                  >
                    <div
                      data-css-14=""
                      data-css-4=""
                    >
                      Lange Gasse 8
                    </div>
                  </ForwardRef>
                  <ForwardRef
                    data-css-14=""
                  >
                    <div
                      data-css-14=""
                      data-css-4=""
                    >
                      CH-4052 Basel
                    </div>
                  </ForwardRef>
                  <ForwardRef
                    data-css-14=""
                  >
                    <div
                      data-css-14=""
                      data-css-4=""
                    >
                      Schweiz
                    </div>
                  </ForwardRef>
                </div>
              </ForwardRef>
              <ForwardRef
                data-css-3=""
                key="Allthings GmbH"
              >
                <div
                  data-css-3=""
                  data-css-4=""
                >
                  <ForwardRef
                    data-css-13=""
                  >
                    <div
                      data-css-13=""
                      data-css-4=""
                    >
                      Allthings GmbH
                    </div>
                  </ForwardRef>
                  <ForwardRef
                    data-css-14=""
                  >
                    <div
                      data-css-14=""
                      data-css-4=""
                    >
                      Unicorn Village
                    </div>
                  </ForwardRef>
                  <ForwardRef
                    data-css-14=""
                  >
                    <div
                      data-css-14=""
                      data-css-4=""
                    >
                      Richardstra√üe 85
                    </div>
                  </ForwardRef>
                  <ForwardRef
                    data-css-14=""
                  >
                    <div
                      data-css-14=""
                      data-css-4=""
                    >
                      D-12043 Berlin
                    </div>
                  </ForwardRef>
                  <ForwardRef
                    data-css-14=""
                  >
                    <div
                      data-css-14=""
                      data-css-4=""
                    >
                      Deutschland
                    </div>
                  </ForwardRef>
                </div>
              </ForwardRef>
              <ForwardRef
                data-css-3=""
                key="Represented by"
              >
                <div
                  data-css-3=""
                  data-css-4=""
                >
                  <ForwardRef
                    data-css-13=""
                  >
                    <div
                      data-css-13=""
                      data-css-4=""
                    >
                      Represented by
                    </div>
                  </ForwardRef>
                  <ForwardRef
                    data-css-14=""
                  >
                    <div
                      data-css-14=""
                      data-css-4=""
                    >
                      Stefan Zanetti
                    </div>
                  </ForwardRef>
                </div>
              </ForwardRef>
              <ForwardRef
                data-css-3=""
                key="Contact"
              >
                <div
                  data-css-3=""
                  data-css-4=""
                >
                  <ForwardRef
                    data-css-13=""
                  >
                    <div
                      data-css-13=""
                      data-css-4=""
                    >
                      Contact
                    </div>
                  </ForwardRef>
                  <ForwardRef
                    alignH="center"
                    direction="row"
                    key="email"
                  >
                    <div
                      data-css-15=""
                    >
                      <ForwardRef
                        alignH="center"
                        alignV="center"
                        dangerouslySetInnerHTML={
                          Object {
                            "__html": "",
                          }
                        }
                        data-css-16=""
                        data-css-1=""
                      >
                        <div
                          dangerouslySetInnerHTML={
                            Object {
                              "__html": "",
                            }
                          }
                          data-css-16=""
                          data-css-4=""
                          data-css-1=""
                        />
                      </ForwardRef>
                      <a
                        href="mailto:support@allthings.me"
                      >
                        support@allthings.me
                      </a>
                    </div>
                  </ForwardRef>
                  <ForwardRef
                    alignH="center"
                    direction="row"
                    key="phone"
                  >
                    <div
                      data-css-15=""
                    >
                      <ForwardRef
                        alignH="center"
                        alignV="center"
                        dangerouslySetInnerHTML={
                          Object {
                            "__html": "",
                          }
                        }
                        data-css-16=""
                        data-css-1=""
                      >
                        <div
                          dangerouslySetInnerHTML={
                            Object {
                              "__html": "",
                            }
                          }
                          data-css-16=""
                          data-css-4=""
                          data-css-1=""
                        />
                      </ForwardRef>
                      <ForwardRef
                        data-css-17=""
                      >
                        <div
                          data-css-4=""
                          data-css-17=""
                        >
                          +49 1511 003 22 54
                        </div>
                      </ForwardRef>
                    </div>
                  </ForwardRef>
                  <ForwardRef
                    alignH="center"
                    direction="row"
                    key="twitter"
                  >
                    <div
                      data-css-15=""
                    >
                      <svg
                        data-css-1=""
                        height="21px"
                        stroke="#7e8c8d"
                        viewBox="0 0 24 24"
                        width="21px"
                      >
                        <path
                          d="M23.407,4.834c-0.814,0.363 -1.5,0.375 -2.228,0.016c0.938,-0.562 0.981,-0.957 1.32,-2.019c-0.878,0.521 -1.851,0.9 -2.886,1.104c-0.827,-0.882 -2.009,-1.435 -3.315,-1.435c-2.51,0 -4.544,2.036 -4.544,4.544c0,0.356 0.04,0.703 0.117,1.036c-3.776,-0.189 -7.125,-1.998 -9.366,-4.748c-0.391,0.671 -0.615,1.452 -0.615,2.285c0,1.577 0.803,2.967 2.021,3.782c-0.745,-0.024 -1.445,-0.228 -2.057,-0.568c-0.001,0.019 -0.001,0.037 -0.001,0.057c0,2.202 1.566,4.038 3.646,4.456c-0.666,0.181 -1.368,0.209 -2.053,0.079c0.579,1.804 2.257,3.118 4.245,3.155c-1.944,1.524 -4.355,2.159 -6.728,1.881c2.012,1.289 4.399,2.041 6.966,2.041c8.358,0 12.928,-6.924 12.928,-12.929c0,-0.198 -0.003,-0.393 -0.012,-0.588c0.886,-0.64 1.953,-1.237 2.562,-2.149Z"
                          fill="none"
                          strokeLinejoin="round"
                          strokeWidth="1"
                        />
                      </svg>
                      <a
                        href="https://twitter.com/allthings_here"
                        rel="noopener"
                        target="_blank"
                      >
                        allthings_here
                      </a>
                    </div>
                  </ForwardRef>
                </div>
              </ForwardRef>
              <ForwardRef
                data-css-3=""
                key="Trade register entry"
              >
                <div
                  data-css-3=""
                  data-css-4=""
                >
                  <ForwardRef
                    data-css-13=""
                  >
                    <div
                      data-css-13=""
                      data-css-4=""
                    >
                      Trade register entry
                    </div>
                  </ForwardRef>
                  <ForwardRef
                    data-css-14=""
                  >
                    <div
                      data-css-14=""
                      data-css-4=""
                    >
                      Register office: Basel
                    </div>
                  </ForwardRef>
                  <ForwardRef
                    data-css-14=""
                  >
                    <div
                      data-css-14=""
                      data-css-4=""
                    >
                      Register number: CH-270.3.014.814-4
                    </div>
                  </ForwardRef>
                </div>
              </ForwardRef>
              <ForwardRef
                data-css-3=""
                key="Tax number (Switzerland)"
              >
                <div
                  data-css-3=""
                  data-css-4=""
                >
                  <ForwardRef
                    data-css-13=""
                  >
                    <div
                      data-css-13=""
                      data-css-4=""
                    >
                      Tax number (Switzerland)
                    </div>
                  </ForwardRef>
                  <ForwardRef
                    data-css-14=""
                  >
                    <div
                      data-css-14=""
                      data-css-4=""
                    >
                      CHE-191.384.657 MWST
                    </div>
                  </ForwardRef>
                </div>
              </ForwardRef>
            </div>
          </ForwardRef>
        </div>
      </ForwardRef>
    </div>
  </ForwardRef>
</Connect(injectIntl(LegalDisclosure))>
`;
